packages/compiler/src/__tests__/__snapshots__/typescript.test.ts.snap

```diff
@@ -425,196 +407,18 @@ Set the noise for the animal",
       },
     ],
     "name": "Dog",
-    "properties": Array [],
+    "properties": [],
     "sourceUrl": "https://github.com/palantir/documentalist/blob/develop/packages/compiler/src/__tests__/__fixtures__/classes.ts#L53",
   },
-  "Food": Object {
```

Comment:
non-exported symbols are no longer documented. I think that's OK.

Reply:
Note that this was actually sort of a _regression_ in https://github.com/palantir/documentalist/pull/144, so removing `Food` here is a good thing.

Reply:
also, we can get this back if we add https://www.npmjs.com/package/typedoc-plugin-missing-exports. so we will have options after this upgrade.